creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -151,6 +152,7 @@ const healthCheck = async (
     longitude,
     databaseConnections,
     databaseSize,
+    databaseIsLocalhost: config.get('dbUrl').search('localhost') > 1,
```

Comment:
don't think this is sufficient since they don't always contain localhost
there seems to have been a change recently to locally map to container name `db` (e.g. check dbUrl on stage-creator-9)

i believe this is only exception so searching for either string prob sufficient? but @dmanjunath maybe knows better

Reply:
i think should be something like 

```suggestion
    databaseIsLocalhost: config.get('dbUrl') === 'postgres://postgres:postgres@db:5432/audius_creator_node' || config.get('dbUrl').includes('localhost')
```

`postgres://postgres:postgres@db:5432/audius_creator_node` is the default url in prod.env in audius-docker-compose https://github.com/AudiusProject/audius-docker-compose/blob/main/creator-node/prod.env#L10